source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,74 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Grouping elastic profile by plugin-id', 3772%>
+
+  This provides an organized view of the elastic profiles making it easier to find a particular elastic agent.
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/elastic-agent-grouping-by-plugin-id.png', alt: 'Group elastic profile by plugin-id' %>
+  </figure>
+
+* <%= link_to_commit 'ca7c3fa232087b67553e7d30bbfb409ba5aab49d', 'Addition of a request processor to allow plugins to query the server info', 3401%>
```

Comment:
Addition of a request processor to allow plugin authors to query information from the GoCD server